src/instructlab/data/generate.py

```diff
@@ -210,6 +211,27 @@ def generate(
         except Exception as exc:
             click.secho(f"Failed to start server: {exc}", fg="red")
             raise click.exceptions.Exit(1)
+
+    # figure out what model is running
+    try:
+        models = ilabclient.list_models(
+            api_base=api_base, http_client=http_client(ctx.params)
+        )
+
+        model = models.data[0].id if models is not None else None
```

Comment:
is it always correct to take the first model in the list? I see the commit message claims:

```
it is not expected that
the user will be serving more than one model at a time anyway
```

But is it always true? Can't we `generate` against a different server, not the one running as `serve`? AFAIU the `api_base` can point to any `OpenAI` compatible endpoint.